deps.edn

```diff
@@ -1,7 +1,38 @@
 {:paths ["src" "resources"]
- :deps {hiccup/hiccup {:mvn/version "1.0.5"}
+ :deps {org.clojure/clojure        {:mvn/version "1.10.3"}
+        lambdaisland/data-printers {:mvn/version "0.0.22"}
```

Comment:
We're at 0.7.47 already for data-printers, maybe run something like [Depot](https://github.com/Olical/depot) on this?

```
clojure -Sdeps '{:deps {olical/depot {:mvn/version "RELEASE"}}}' -M -m depot.outdated.main --write
```